.github/workflows/alpha-release.yml

```diff
@@ -16,77 +16,82 @@ jobs:
       changelog: ${{ steps.changelog.outputs.changelog }}
       release_id: ${{ steps.release.outputs.id }}
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
         with:
           fetch-depth: 0
           token: ${{ secrets.PRIVATE_TOKEN }}
 
-      - uses: actions/setup-node@v3
+      - name: Set up version.json
```

Comment:
If this is used by the changelog, maybe move it so we do the installations first, and then do this just before the changelog step. I stared at this for a long time trying to figure out why node would need this...

Reply:
Fixed.